sklearn/tests/test_multioutput.py

```diff
@@ -85,3 +89,74 @@ def test_multi_target_sample_weights():
 
     X_test = [[1.5,2.5,3.5], [3.5,4.5,5.5]]
     assert_almost_equal(rgr.predict(X_test), rgr_w.predict(X_test))
+
+# Creates a multi-target data set using the iris data set
+# X : The iris predictor data.
+# y : A multi-target(150x3) array generated from the original response data
+
+# Import the data
+iris = datasets.load_iris()
+# create a multiple targets by randomized shuffling and concatenating y.
+X = iris.data
+y1 = iris.target
+y2 = shuffle(y1, random_state=1)
+y3 = shuffle(y1, random_state=2)
+# concatenate the array and transpose
+y = np.vstack((y1, y2, y3)).T
+n_samples, n_features = X.shape
+n_outputs = y.shape[1]
+n_classes = 3
+
+
+def test_multi_target_fit_with_random_forest():
+    # test if multi_target initializes correctly with random forest and fit
+    # assert predictions work as expected for predict, prodict_proba and score
+
+    forest = RandomForestClassifier(n_estimators=100, random_state=1)
```

Comment:
You should also write a test for `sample_weights`, see the regression tests for inspiration.